compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Conversions.scala

```diff
@@ -28,7 +28,7 @@ final class Conversions(
   // We need the mapping for converting error message so we manually add it here.
   private val ptxCoidToNodeId = machine.ptx.nodes
     .collect {
-      case (nodeId, node: N.NodeCreate.WithTxValue[V.ContractId]) =>
+      case (nodeId, node @ N.NodeCreate(_, _, _, _, _, _)) =>
```

Comment:
(Follows up #206, #643.) Perhaps the definition of `Transaction.Value` should change, instead? Or,

```suggestion
      case (nodeId, node: N.NodeCreate[_, V.ContractId, _]) =>
```

Reply:
In the current state we should have to add a `WithPartialTransactionValue` type. 
I will see later if it is worthwhile to add it.